start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ "$NODE_ENV" == "production" ] ; then
+  yarn start:prod
+else
+  yarn start
```

Comment:
Reminder : find a way to develop into the nest docker container(with a shared volume containing the source code ?)